python/init.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# XCFun, an arbitrary order exchange-correlation library
+# Copyright (C) 2018 Ulf Ekström and contributors.
+#
+# This file is part of XCFun.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# For information on the complete list of contributors to the
+# XCFun library, see: <https://xcfun.readthedocs.io/>
+#
+
+from .xcfun import (Functional, xc_eval, xc_eval_setup, xc_new_functional, xcfun_splash, xcfun_version)
```

Comment:
Can this be changed to `from .xcfun import *`? I know this is not nice, but PyADF uses not only the functions defined in xcfun.py, but also some of the swig-generated ones in xcfun_swig.py are used directly.

Reply:
But can't we list them explicitly somehow? The `import *` is evil :-)

Reply:
Generally, I would agree, but here the situation seems a bit different to me. 
xcfun_swig.py is auto-generated and exposes the full API of XCFun (with some small modifications and additions). I think (at least for now) this should be fully available in Python. But if you disagree, I will have a closer look to find out which functions from xcfun_swig.py are should be exposed and which ones should be hidden.

Reply:
Lets use `import *` for now. Before switching to something new we should get back to this and discuss what the Python API should look like and separate public and private parts.

Reply:
This is done.